src/main/java/com/royalrangers/controller/SubscribeController.java

```diff
@@ -30,7 +38,11 @@ public ResponseResult subscribe(@RequestBody Map<String, Object> params) {
     }
 
     @DeleteMapping
-    public ResponseResult unsubscribe(@RequestBody  Map<String, Object> params) {
+    @ApiOperation(value = "Delete subscriber")
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = "json", value = "Subscriber's email. Example: {\"email\": \"test@test.com\"}", paramType = "body", required = true)
```

Comment:
We need to think how to describe parameters in swagger in case we accept **Map<String, Object> params**, in order to avoid such an expressions **Example: {\"email\": \"test@test.com\"}**